core/src/main/java/com/datastax/oss/driver/internal/core/connection/ConstantReconnectionPolicy.java

```diff
@@ -19,16 +19,23 @@
 import com.datastax.oss.driver.api.core.config.DriverConfigProfile;
 import com.datastax.oss.driver.api.core.connection.ReconnectionPolicy;
 import com.datastax.oss.driver.api.core.context.DriverContext;
+import com.datastax.oss.driver.api.core.metadata.Node;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.time.Duration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** A reconnection policy that waits a constant time between each reconnection attempt. */
 public class ConstantReconnectionPolicy implements ReconnectionPolicy {
```

Comment:
This implementation interrogates `DefaultDriverOption.RECONNECTION_BASE_DELAY` internally. Should we mention in the class javadocs which settings can affect this policy's behavior so that users know which settings to customize when using this policy? IOW, users trying to figure out how to use some specific policy by simply reading the javadocs will have trouble finding how to configure it properly. (This question is btw also valid for the exponential policy below, and for all policies in general).

Reply:
👍 I'll revisit the policies to make sure that they all show a small config example.